### PR TITLE
Update CODEOWNERS for security assessments.

### PR DESCRIPTION
**Pull request by lumjjb**
Adding @JustinCappos to CODEOWNERS for security assessments.

### CODE REVIEW

**Comment by lumjjb** [comment]
@JustinCappos  Require your approval on this.

**Comment by lumjjb** [comment]
Need a code owner approval : @ultrasaurus @dshaw @pragashj

File: CODEOWNERS [hunk 1]
```diff
@@ -10,7 +10,7 @@
 *       @ultrasaurus @dshaw @pragashj
 
 # Security assessments (TODO: Approval required by PR)
```
Comment (ultrasaurus) [inline CODEOWNERS]: Does this imply removing the TODO?

**Comment by ultrasaurus** [comment]
See question about TODO -- approving so you can merge after you answer that questions (either in PR or comment)

File: CODEOWNERS (hunk 1 above)
Comment (lumjjb) [reply CODEOWNERS]: Yup! I'll remove it.

**Comment by lumjjb** [comment]
>Approve -- since this is already documented in /roles (e.g. not a change in process, just supporting logistics of already documented process, I think single chair approval is sufficient)

Document github settings are consistent with this

Screen Shot 2020-03-25 at 11 25 50 AM

**Pull request merged**